src/app.js

```diff
@@ -1,20 +1,64 @@
-const express = require('express');
-const app = express();
+require('dotenv').config()
+const express = require('express')
+const app = express()
+const morgan = require('morgan')
+const bodyParser = require('body-parser')
 const cors = require('cors')
-const favicon = require('express-favicon');
-const logger = require('morgan');
+const favicon = require('express-favicon')
+const mainRouter = require('./routes/mainRouter.js')
+const helmet = require('helmet')
+const session = require('express-session')
+const passport = require('passport')
+const mongoose = require('mongoose')
+const asyncErrors = require('express-async-errors')
+const cookieParser = require('cookie-parser')
+const rateLimit = require('express-rate-limit')
+const xss = require('xss')
+const mongoSanitize = require('mongo-sanitize')
+const limiter = rateLimit({
```

Comment:
should I delete these dependencies from lines 15-18 ?

Reply:
I don't think you should delete, you just need to add `app.use(limiter)` somewhere below this in the file so that the limiting will actually be applied for requests going through the Express server 🙂

Reply:
added  commit 78f5c188050a31f6e0827daf03e12d01c88e4e74
Thank you very much!